extensions/yubikey/yubikey.extension.php

```diff
@@ -50,9 +50,9 @@ function store_get_uid_from_yubikey($otp) {
 
 		if (isset($test_user['auth_method']) && $test_user['auth_method'] == 'YUBIKEY') {
 			if (isset($test_user['yubikey']) && is_array($test_user['yubikey'])) {
-				if (isset($test_user['yubikey']['key_id']) && $test_user['yubikey']['key_id'] === $keyID) {
+				if (isset($test_user['yubikey']['key_id']) && in_array($keyID, $test_user['yubikey']['key_id'], true)) {
```

Comment:
You have a typo in the commit message of this commit. It should be `for each account`, but it `for ach account`.